src/app/containers/ProposalDetailsPage/index.tsx

```diff
@@ -171,65 +202,86 @@ export function ProposalDetailsPage() {
             </div>
           </div>
           <div
-            className={`flex justify-center xl:mt-20 mt-10
+            className={`flex flex-col
             ${proposalLoading && 'skeleton'}`}
           >
-            <div className="mr-10 text-right">
-              <span className="xl:text-3xl text-xl font-semibold leading-5 tracking-normal">
-                {(votesForProgressPercents || 0).toLocaleString(undefined, {
-                  minimumFractionDigits: 2,
-                  maximumFractionDigits: 2,
-                })}
-                %
-              </span>
-              <Tooltip2
-                className={Classes.TOOLTIP2_INDICATOR}
-                minimal={true}
-                content={
-                  <p className="text-white text-sm tracking-normal">
-                    {numberFromWei(data?.forVotes || 0)} votes
-                  </p>
-                }
-                placement="top"
-              >
-                <p className="xl:text-lg text-sm font-light tracking-normal">
-                  {kFormatter(numberFromWei(data?.forVotes || 0))} votes
-                </p>
-              </Tooltip2>
+            <div className="xl:mt-20 mt-10 mb-6 text-center">
+              <div className="mb-2">Percentage Voted:</div>
+              {!totalVotingPowerLoading ? (
+                <>
+                  {`${voteRatio}% = ${kFormatter(
+                    numberFromWei(totalVotedPower || 0),
+                  )} / ${kFormatter(numberFromWei(totalVotingPower || 0))}`}
+                </>
+              ) : (
+                <div
+                  className="skeleton mx-auto"
+                  style={{ width: '200px', height: '40px' }}
```

Comment:
Why do you have this as inline style?

Reply:
Refactored into styled component as discussed